build.sbt

```diff
@@ -35,7 +35,8 @@ val circeVersion = "0.8.0"
 val commonsIoVersion = "2.5"
 val commonsMath3Version = "3.6.1"
 val csvVersion = "0.2.0"
-val elasticsearchVersion = "2.1.0"
+val elasticsearch2Version = "2.1.0"
+val elasticsearch5Version = "5.5.0"
```

Comment:
@Alevy01 +1 on using 5.5.0 was about to suggest the same. While you are at it and if don't mind, we could bump ES 2.1.0 to 2.4.5